content/articles/dns-hosting.md

```diff
@@ -31,8 +30,7 @@ If you want to use DNSimple to manage the DNS of an existing domain, you have tw
 
 If you don't have a domain yet, you can [register a domain](https://dnsimple.com/tlds) with us, and subscribe to a DNS hosting plan to manage the domain with us.
```

Comment:
What about also adding they can transfer a domain to us?

> If you don't have a domain yet, you can either [register](https://dnsimple.com/tlds)) or [transfer](https://support.dnsimple.com/articles/domain-transfer/) a domain to us and then subscribe to a DNS hosting plan to manage it with us.

Reply:
Who can say? That's in the existing article. We can just say 'plan' and link the pricing page

Reply:
I thought about it, and you're right it t's not necessary. Plus, we already covered it in the second point.

Reply:
Hang on, don't you have to be subscribed to a plan to register a domain in the first place? Would it make more sense to say 
"If you don't have a domain yet, you can [sign up](plan page link) to [register a domain](https://dnsimple.com/tlds) and activate the zone to manage your DNS with us." ?

Reply:
You are totally correct, and the way you formulated the sentences makes more sense. Maybe add "or log in" as well? 

> If you don't have a domain yet, you can [sign up](plan page link) or log in to [register a domain](https://dnsimple.com/tlds) and activate the zone to manage your DNS with us.